sidebar.js

```diff
@@ -164,6 +164,21 @@ const sidebars = {
       items: [{ type: "autogenerated", dirName: "scripting/db" }],
     },
   ],
+  miscSidebar: [
+    {
+      type: "category",
+      label: "Miscellaneous",
```

Comment:
```suggestion
      label: "Syntax",
```

I checked with a couple volunteers and we think making a new sidebar category was the right idea. Though it would help players to be more descriptive, so this might be better suited as a category for "Syntax".

E.g.: (Feel free to change this wording)

### Hackmud Syntax

This is a list of all custom syntax that hackmud uses for behavior and formatting.